app/charts/map/map.tsx

```diff
@@ -175,6 +187,18 @@ export const MapComponent = () => {
     mapNodeRef.current = mapRef;
   };
 
+  const refresh = () => {
```

Comment:
refresh or reset ? Can you add a small comment here, it is not immediately clear to me what this should do.